corehq/apps/users/views/mobile/users.py

```diff
@@ -293,6 +293,16 @@ def page_context(self):
             make_form_readonly(self.form_user_update.user_form)
             make_form_readonly(self.form_user_update.custom_data.form)
 
+        location_info = get_user_location_info(
+            domain=self.domain,
+            user_location_ids=self.editable_user.assigned_location_ids,
+            user_id=self.editable_user.user_id
+        )
+
+        edit_groups_permission = self.request.couch_user.has_permission(self.domain, 'edit_groups')
```

Comment:
Could rename `edit_groups_permission` to `can_edit_groups`.

Reply:
https://github.com/dimagi/commcare-hq/pull/33184/commits/29621c6677ba08cb7e019e4d374dc4ecf94d0634